utils/mock-data-loader/database/user-repository.js

```diff
@@ -23,14 +23,19 @@ const issueValid2faJWT = (user, sessionIdentifier) => {
 
 const createLoggedInUserSession = async (user) => {
   try {
+    console.info('Hello 123123');
+    console.info(process.env.MONGODB_URI);
     const userCollection = await db.getCollection('users');
+    console.info(userCollection);
+    console.info('7784778478478478');
     const userFromDatabase = await userCollection.findOne({ username: { $eq: user.username } }, { collation: { locale: 'en', strength: 2 } });
 
     const sessionIdentifier = crypto.randomBytes(32).toString('hex');
     const token = issueValid2faJWT(userFromDatabase, sessionIdentifier);
     await userCollection.updateOne({ _id: { $eq: userFromDatabase._id } }, { $set: { sessionIdentifier } });
     return `Bearer ${token}`;
   } catch (e) {
+    console.error('Failed to create %O', e);
     throw new Error(`Failed to create logged in user session for user: ${user.username}: ${e}`);
```

Comment:
Will this just log `[Object.object]`? I'd like to start using `cause` for passing through the cause of the error ie


```suggestion
    throw new Error(`Failed to create logged in user session for user: ${user.username}`, {cause: e});
```

Reply:
This was some logging I didn't actually mean to commit. FWIW no it logged the full error but I'll delete!

Reply:
How odd , behaves differently in console in browser and on my local machine on an empty project 
![image](https://github.com/UK-Export-Finance/dtfs2/assets/47955140/fd631337-e03d-489f-8629-6d18d8539376)
! I'll file that along with how deep cloning works on this proj.